composer.json

```diff
@@ -66,7 +66,22 @@
         "refresh": [
             "rm -rf composer.lock vendor html/core html/modules/contrib html/profiles/contrib html/themes/contrib",
             "@composer update -W --ansi"
-        ]
+        ],
+        "docker-up": [
+            "docker compose -f docker-compose.nginx.yml -p nginx up -d",
+            "docker compose up -d --remove-orphans"
+        ],
+        "docker-shell": "test -f \".env\" || exit 1; export $(egrep -v '^#' .env | xargs); test -f \".env\"; sh -c \"docker exec -ti ${PROJECT_NAME}_web bash\"",
+        "docker-stop": "docker-compose stop",
+        "os-install":  "test -f \".env\" || exit 1; export $(egrep -v '^#' .env | xargs); test -f \".env\"; sh -c \"docker exec -i ${PROJECT_NAME}_web bash /var/www/scripts/social/install/install_script.sh\"",
+        "phpstan": "vendor/bin/phpstan analyse -c html/profiles/contrib/social/phpstan.neon --memory-limit=-1",
+        "phpstan-baseline": "vendor/bin/phpstan analyse -c html/profiles/contrib/social/phpstan.neon --memory-limit=-1 --generate-baseline",
```

Comment:
Not surewe want to shortcut this? Since this shouldn't really be happening anymore? ^^